scripts/common

```diff
@@ -86,6 +86,18 @@ else
   )
 fi
 
+if [[ -z "${RUST_HOST_TARGET:-default}" ]]; then
+  if [[ "${OSTYPE}" == "darwin"*  ]]; then
+    build_target="x86_64-apple-macosx10.7.0"
+  else
+    build_target="x86_64-unknown-linux-gnu"
+  fi
+else
+  build_target="${RUST_HOST_TARGET:-default}"
+fi
+
```

Comment:
Is it possible to do this with fewer variables?  Also, it would be nice if the environment variable (`RUST_HOST_TARGET`) got used in the build commands, but I guess that might mean no `readonly`